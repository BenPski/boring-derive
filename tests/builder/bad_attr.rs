use boring_derive::Builder;

#[derive(Builder)]
struct Example {
    #[builder(not_real)]
    item: usize,
}

fn main() {}
