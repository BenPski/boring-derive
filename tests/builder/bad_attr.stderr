error: unknown builder field attribute: `not_real`
 --> tests/builder/bad_attr.rs:5:15
  |
5 |     #[builder(not_real)]
  |               ^^^^^^^^
